config/prod/toil-shared-cluster-role-and-policy.yaml

```diff
@@ -0,0 +1,2 @@
+template_path: remote/toil-cluster-role-and-policy.yaml
+stack_name: toil-cluster-shared-role
```

Comment:
there's a `ClusterName` parameter in templates/toil-cluster-role-and-policy.yaml file.  you need to pass in something for that or alternatively remove the parameter.